src/pilot/templates/nic-configs/dvr_4_port/compute.yaml

```diff
@@ -90,14 +90,25 @@ parameters:
   EC2MetadataIp: # Override this via parameter_defaults
     description: The IP address of the EC2 metadata server.
     type: string
-
-  # CHANGEME: If you wish to use different bonding modes for different roles,
-  # then specify the appropriate bonding options in the "default" line below
-  # for the compute role, otherwise ignore this entry.
-  BondInterfaceOptions:
-    default: ''
-    description: The bonding_options string for the bond interface. Set things
-                 like mode=802.3ad using this option.
+  ComputeBond0Interface1:
+    default: 'em1'
+    description: Bond 0 interface 1 name
+    type: string
+  ComputeBond0Interface2:
+    default: 'p1p1'
+    description: Bond 0 interface 2 name
+    type: string
+  ComputeBond1Interface1:
+    default: 'em2'
+    description: Bond 1 interface 1 name
+    type: string
+  ComputeBond1Interface2:
+    default: 'p1p2'
+    description: Bond 1 interface 2 name
+    type: string
+  ComputeBondInterfaceOptions:
+    default: '802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1'
```

Comment:
The above line should be "default: 'mode=802.3ad ...'".  This is incorrect in the existing nic configs.  I will submit a PR fixing them.